src/index.html

```diff
@@ -9,6 +9,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="Moyo online store logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a class="is-active" href="#apple">Apple</a>
```

Comment:
```suggestion
            <a class="nav__link is-active" href="#apple">Apple</a>
```
Створи для посилань клас